src/main/java/com/royalrangers/controller/UserController.java

```diff
@@ -136,6 +140,10 @@ public ResponseResult rejectUser(@PathVariable("userId") Long id) {
     @PutMapping("/update/temp")
     @ApiOperation(value = "Update user data (for current user)")
     public ResponseResult updateTempUser(@RequestBody UserUpdateDto update) {
+        if (userService.isTempUserExist(userService.getAuthenticatedUserId())) {
+            log.info("TempUser already exists");
+            return ResponseBuilder.fail("Waiting to confirm existing update by admin");
+        }
```

Comment:
Don't remember when we discussed to add this verification. 
Not sure it is needed here. 
As for me, it's ok if user wants to change his info before it's finally approved by an admin.